### PR TITLE
Add Covifenz

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Example of a VLP!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/22.vaccines.md [hunk 1]
```diff
@@ -316,7 +315,37 @@ In a phase III randomized, observer-blinded, placebo-controlled clinical trial i
...
 
-<!--Add summary of Covifenz https://covid19.trackvaccines.org/vaccines/26/ as an example of VLP-->
+The leading example of a VLP approach applied to COVID-19 comes from Covifenz, a VLP vaccine developed by Canadian company Medicago [@doi:10.1503/cmaj.1095992].
+This vaccine was developed using plant-based VLP technology [@url:https://medicago.com/en/our-products/our-vaccines/covifenz-covid-19-vlp-vaccine] that the company had been investigating in order to develop a high-throughput quadrivalent VLP platform to provide protection against influenza [@doi:10/gjn28x].
+The approach utilizes _Nicotiana benthamiana_, an Australian relative of the tobacco plant, as an upstream bioreactor [@doi:10/gjn28x; @doi:10.1038/s41591-021-01370-1; @doi:10.1038/s41591-021-01370-1].
+Specifically, the _S_ gene from SARS-CoV-2 in its prefusion conformation is inserted into a bacterial vector (_Agrobacterium tumefaciens_) that then infects the tobacco plant cells [@doi:10/gjn28x; @doi:10.1038/s41591-021-01370-1].
```
Comment (RLordan) [inline content/22.vaccines.md:321]: Do you still refer to it as tobacco if it is only a relation to tabocco?

File: content/22.vaccines.md [hunk 2]
```diff
@@ -316,7 +315,37 @@ In a phase III randomized, observer-blinded, placebo-controlled clinical trial i
...
+Expression of the S glycoprotein causes the production of VLPs composed of S trimers anchored in a lipid envelope that accumulate between the plasma membrane and the cell wall of the plant cell [@doi:10.1038/s41591-021-01370-1].
+Because these VLPs do not contain the SARS-CoV-2 genome, they offer similar advantages to while mitigating the risks of whole-virus vaccines [@doi:10/gjn28x; @doi:10.1038/s41591-021-01370-1].
+
+In the Phase I study, Covifenz was administered to 180 Canadian adults 18-55 years old as two doses, 21 days apart, with three different dosages evaluated [@doi:10.1038/s41591-021-01370-1].
+This study reported that when the VLPs were administered with an adjuvant, the vaccine elicited a neutralizing antibody that was significantly (approximately 10 times) higher than that in convalescent sera  [@doi:10.1038/s41591-021-01370-1].
+They also reported a cellular immune response was induced.
+Based on these findings, phase II and III trials began.
+
+The findings of the phase III trial were published a few months later in the _New England Journal of Medicine_ [@url:10.1056/NEJMoa2201300].
```
Comment (RLordan) [inline content/22.vaccines.md:330]: This was recent, good spot!

File: content/22.vaccines.md [hunk 3]
```diff
@@ -316,7 +315,37 @@ In a phase III randomized, observer-blinded, placebo-controlled clinical trial i
...
+This study examined 24,141 adults assigned to the treatment and control conditions at a 1:1 ratio between March and September of 2021.
+Participants were recruited from countries in North and South America, as well as the United Kingdom.
+Approximately 10,000 individuals in each condition completed both doses and were evaluated in the per-protocol population.
+Data submitted by Medicago and GSK to Health Canada, the vaccine was approved for use in adults ages 18 to 65 in February 2022 [@url:https://www.gsk.com/en-gb/media/press-releases/medicago-and-gsk-announce-the-approval-by-health-canada-of-covifenz].
```
Comment (RLordan) [inline content/22.vaccines.md:334]:
```suggestion
Data was submitted by Medicago and GSK to Health Canada and the vaccine was approved for use in adults ages 18 to 65 in February 2022 [@url:https://www.gsk.com/en-gb/media/press-releases/medicago-and-gsk-announce-the-approval-by-health-canada-of-covifenz].
```

File: content/22.vaccines.md [hunk 4]
```diff
@@ -316,7 +315,37 @@ In a phase III randomized, observer-blinded, placebo-controlled clinical trial i
...
+
+Plant-based expression systems such as this are relatively new [@doi:10.1038/s41591-021-01370-1] but are likely to offer unparalleled feasibility at scale given the speed and low-cost associated with the platform [@doi:10.1111/j.1467-7652.2008.00384.x].
+Additionally, it can be stored at 2 to 8&#176;C.<!--Check this renders correctly as degree symbol-->
+However, the worldwide footprint of Covifenz, and of VLP-based technologies against SARS-CoV-2 broadly, remains small, with only {{owid_VLP_count}} VLP vaccine has been approved for distribution in {{owid_VLP_countries}} countries (Figure @fig:vlp-distrib).<!--Grammar needs update if anyone outside of Canada approves or if another VLP becomes active-->
```
Comment (RLordan) [inline content/22.vaccines.md:338]: This confused me grammatically but I see your note, so ignoring.

File: content/22.vaccines.md [hunk 5]
```diff
@@ -327,6 +356,14 @@ All moderate to severe cases of COVID-19 occurred in the placebo group.
 Additionally, in both trials, the vaccine was found to be well-tolerated [@doi:10.1101/2021.10.05.21264567; @doi:10.1056/NEJMoa2107659].
 The conclusions of both trials indicate that the NVX-CoV2373 vaccine is safe and effective against COVID-19, and those who received this vaccine through the trials are considered fully vaccinated [@url:https://ir.novavax.com/Novavax-Statement-on-UK-and-Mexico-Phase-3-Clinical-Trial-Participants-Considered-Fully-Vaccinated-in-the-US].
 
+Similarly, Covifenz was reported to be 71% effective in preventing COVID-19 in the per-protocol analysis [@doi:10.1056/NEJMoa2201300].
+Efficacy was only slightly slower in the intention-to-treat group at 69%.
```
Comment (RLordan) [inline content/22.vaccines.md:360]:
```suggestion
Efficacy was only slightly lower in the intention-to-treat group at 69%.
```

File: content/22.vaccines.md [hunk 6]
```diff
@@ -327,6 +356,14 @@ All moderate to severe cases of COVID-19 occurred in the placebo group.
...
+Prevention of moderate-to-severe disease was estimated at 78.8% in the intention-to-treat group.
+Over 24,000 participants were included in the safety analysis, which reported that 92.3% of vaccine recipients reported local adverse events compared to 45.5% of placebo recipients, with rates for systemic adverse events at 87.3% and 65.0%, respectively.
+However, the AEs reported were generally mild to moderate.
```
Comment (RLordan) [inline content/22.vaccines.md:363]: Are we abbreviating to AEs everywhere?

**Review by RLordan — APPROVED** [review]
This vaccine is really cool, nice review of it.

File: content/22.vaccines.md (hunk 1 above)
Comment (rando2) [reply content/22.vaccines.md:321]: You're right, it's called "benthi"!
```suggestion
Specifically, the _S_ gene from SARS-CoV-2 in its prefusion conformation is inserted into a bacterial vector (_Agrobacterium tumefaciens_) that then infects the plant cells [@doi:10/gjn28x; @doi:10.1038/s41591-021-01370-1].
```

File: content/22.vaccines.md (hunk 4 above)
Comment (rando2) [reply content/22.vaccines.md:338]:
```suggestion
However, the worldwide footprint of Covifenz, and of VLP-based technologies against SARS-CoV-2 broadly, remains small, with only {{owid_VLP_count}} VLP vaccine approved for distribution in {{owid_VLP_countries}} countries (Figure @fig:vlp-distrib).<!--Grammar needs update if anyone outside of Canada approves or if another VLP becomes active-->
```
It was wrong! But obfuscated by all the variables. Good catch!

File: content/22.vaccines.md (hunk 6 above)
Comment (rando2) [reply content/22.vaccines.md:363]: I'm going to do that on my next read through, somehow we need to cut a lot of words still, so that's one way, haha

Comment (RLordan) [reply content/22.vaccines.md:363]: Ha! Very good

**Pull request merged**